script.js

```diff
@@ -0,0 +1,65 @@
+const titleInput = document.getElementById('bookTitle');
+const authorInput = document.getElementById('bookAuthor');
+const addBook = document.getElementById('add');
+const list = document.getElementById('list');
+
+function Book(id, title, author) {
+  this.id = id;
+  this.title = title;
+  this.author = author;
+}
+
+let BookArray = [];
+
+function AddBook(id, title, author) {
+  BookArray.push(new Book(id, title, author));
+}
+
+function DispalyTheBook() {
+  for (let i = 0; i < BookArray.length; i += 1) {
+    const title = document.createElement('li');
+    const author = document.createElement('li');
+    const remove = document.createElement('Button');
+    const titleLi = document.createTextNode(BookArray[i].title);
+    const authorLi = document.createTextNode(BookArray[i].author);
+    remove.textContent = 'remove';
+    remove.value = i;
+    remove.id = i;
+    title.id = `title${i}`;
+    author.id = `author${i}`;
+    const removeButton = document.getElementById(i);
+    if (removeButton == null) {
+      title.appendChild(titleLi);
+      author.appendChild(authorLi);
+      list.appendChild(title);
+      list.appendChild(author);
+      list.appendChild(remove);
+    }
+  }
+}
+
+function removeBookList() {
+  for (let i = 0; i < BookArray.length; i += 1) {
+    if (document.getElementById(BookArray[i].id) != null) {
+      // eslint-disable-next-line no-loop-func
+      document.getElementById(BookArray[i].id).addEventListener('click', () => {
+        function isId(value) {
+          return value.id !== BookArray[i].id;
+        }
+        document.getElementById(`title${i}`).remove();
```

Comment:
**Kindly fix this error:**
- When i try to remove an item from the list. This error is displayed on the chrome console:
<img width="1440" alt="Screen Shot 2021-09-13 at 19 33 29" src="https://user-images.githubusercontent.com/22258274/133174726-c592fcc4-175c-4914-bc18-6e7122e98937.png">